server/api/incidents.go

```
@@ -1327,6 +1327,42 @@ func parseIncidentsFilterOptions(u *url.URL) (*app.IncidentFilterOptions, error)
 
 	playbookID := u.Query().Get("playbook_id")
 
+	activeGTEParam := u.Query().Get("active_gte")
+	if activeGTEParam == "" {
+		activeGTEParam = "0"
+	}
+	activeGTE, err := strconv.ParseInt(activeGTEParam, 10, 64)
+	if err != nil {
```

Given we default to `0` anyway, should we simplify each block to the following, and effectively just ignore the error of a garbage value?
```
activeGTE, _ := strconv.ParseInt(u.Query().Get("active_gte"), 10, 64)
```